hexrd/sampleOrientations/sampleRFZ.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+from hexrd.utils.decorators import numba_njit_if_available
+from hexrd.sampleOrientations.conversions import cu2ro, ro2qu
+from hexrd.sampleOrientations.rfz import insideFZ
+from hexrd import constants
+
+if constants.USE_NUMBA:
+    from numba import prange
+else:
+    prange = range
+
+@numba_njit_if_available(cache=True, nogil=True)
```

Comment:
```suggestion
@numba_njit_if_available(cache=True, nogil=True, parallel=True)
```
I believe that in order to use `prange`, you must pass `parallel=True`.

However, it is possible that the parallel version of this could be slower than the serial version, since some of the numpy functions might already be parallel underneath. So if this does get changed, be sure to test it and see the performance difference.